### ISSUE TITLE
[Short summary of the question]More details for using CleanLearning with custom models

### ISSUE THREAD

**Issue by heng3366** [cleanlab/cleanlab#801]
eg：
model_skorch   = NeuralNetClassifier(model, device="cuda", max_epochs=20, batch_size=512)
quesitions:
1、Do the optional parameters of the NeuralNetClassifier align with those used for fine-tuning the SOTA model（my finetune model）?
2、If the cross-validation accuracy decreases by a few points, does it affect the final results of finding label issues?

**Comment by jwmueller** [comment]
Re 1: Yes, we recommend specifying the same optional parameter values as you're using for your model.

Re 2: Yes, worse cross-validation accuracy generally means your results will be worse. You should generally try to get the best cross-validation accuracy that you are able to achieve before running find_label_issues
